test/e2e/cluster.sh

```diff
@@ -112,6 +112,22 @@ docker run --rm \
 -e GINKGO_SKIP="${GINKGO_SKIP}" \
 "${DEV_IMAGE}" make test-kubernetes || exit 1
 
+if [ "${GET_CLUSTER_LOGS}" = "true" ]; then
```

Comment:
So, this if statement needs to be incorporated into the if statement below:

`if [ "${UPGRADE_CLUSTER}" = "true" ] || [ "${SCALE_CLUSTER}" = "true" ] || [ -n "$ADD_NODE_POOL_INPUT" ]; then`

e.g.:

`if [ "${UPGRADE_CLUSTER}" = "true" ] || [ "${SCALE_CLUSTER}" = "true" ] || [ -n "$ADD_NODE_POOL_INPUT" ] || [ "${GET_CLUSTER_LOGS}" = "true" ]; then`

We can't put it above that, because the `$RESOURCE_GROUP` and `$REGION` variables haven't yet been defined.

Reply:
sorry, my bad, I was rushing